compiler/testData/diagnostics/tests/extensions/contextReceivers/overloadPriority.kt

```diff
@@ -0,0 +1,12 @@
+class Context
+
+context(Context)
+fun f() {}
+
+fun f() {}
+
+fun test() {
+    with(Context()) {
+        f()
```

Comment:
Please add some non-trivial return type to `f`, so we may check which one function is chosen

Reply:
Added